PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+# Pull Request Template
+
+---
+## Protocol Information
+
+- **Name:**
+- **Twitter Link:**
+- **Website Link:**
+- **Github:**
```

Comment:
Is `Github` provided in an account or a URL?

Reply:
It seems "GitHub" isn't very intuitive for you. Should we change it to "GitHub Link"?
cc @zodahu

Reply:
Updated